diff --git a/pyproject.toml b/pyproject.toml
index 2884a66..6d27e5e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,7 +31,7 @@ dependencies= [
     # NOTE: the tiledbsoma version must be >= to the version used in the Census builder, to
     # ensure that the assets are readable (tiledbsoma supports backward compatible reading).
     # Make sure this version does not fall behind the builder's tiledbsoma version.
-    "tiledbsoma~=1.11.4",
+    "tiledbsoma>=1.11.4",
     "anndata",
     "numpy>=1.21,<2.0",
     "requests",
